readme.md

```diff
@@ -77,7 +80,7 @@ build:
     $ mkdir build 
     $ cd build
     $ cmake .. 
-    $ make -j$(nproc)
+    $ make -j
```

Comment:
This is basically a fork bomb

Reply:
How so?

Reply:
`make -j` without a number spawns as many compilation processes as it can.